### DISCUSSION TITLE
How to use the latest versions

### DISCUSSION THREAD

**Discussion by haritowa** [terra-farm/terraform-provider-virtualbox#158] [Support] [answered]
Is there any recommended way to use the latest version(master)? The latest release both on GitHub and Registry is from Sep 2021

**Comment by VoyTechnology** [comment, marked as answer]
Thanks to the improved contributing guide by @nikstuckenbrock, [Debugging](https://github.com/terra-farm/terraform-provider-virtualbox/blob/main/CONTRIBUTING.md#debugging) section in `CONTRIBUTING.md` should have all the info you will need.
